com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -72,6 +72,21 @@ def transform(self, func):
         op.addInputPort(outputPort=self.oport)
         oport = op.addOutputPort()
         return Stream(self.topology, oport)
+     
+    def multiTransform(self, func):
+        """
+        Transforms each tuple from this stream into 0 or more tuples using the supplied function.
+        For each tuple on this stream, the returned stream will contain all non-None tuples in
```

Comment:
Yes, the function should return an iterable.  Added the following in the comment:
The supplied function must return an iterable, otherwise a TypeError is raised.